nagios/opsmatic-nagios.rb

```diff
@@ -74,17 +74,13 @@
 event = {
   :timestamp    => data[:notification_time],
   :source       => "nagios",
-  :subject_type => "hostname",
   :type         => "notifications/nagios",
   :data         => {},
+  :scopes       => {
+    :hostname => data[:host][:name] ? options[:name_pref] == "name" : data[:host][:alias]
```

Comment:
this ternary is backwards.

please try to test this in our own nagios if you can